concepticondata/conceptlists.tsv

```diff
@@ -467,4 +467,5 @@ DellaRosa-2010-417	Della Rosa, Pasquale A. and Catricà, Eleonora and Vigliocco,
 Ray-1889-78	Ray, Sidney H.	1889		78		English	Oceanic languages	https://zenodo.org/records/2403292	Ray1889		This list of 78 words was chosen to show the 'variation' in some Oceanic languages compared to Baki.	302-303	
 Schauenburg-2015-858	Schauenburg, Gesche and Ambrasat, Jens and Schröder, Tobias and Von Scheve, Christian and Conrad, Markus	2015		858	ratings	German	German	https://doi.org/10.3758/s13428-014-0494-7	Schauenburg2015		This list contains ratings of valence, arousal, potency, authority and community for 858 German words. Participants were native German speakers recruited through university mailing lists.	720-735
 Janschewitz-2008-460	Janschewitz, Kristin	2008		460	ratings	English	English	https://doi.org/10.3758/BRM.40.4.1065	Janschewitz2008		This list of 460 English taboo, emotionally valenced, and emotionally neutral words was rated for frequency, inappropriateness, valence, arousal, and imageability by 78 native-English-speaking college students.	1065-1074
-Tryon-1995-1310	Tryon, Darrell T.	1995		1310	questionnaire	English	Austronesian languages	https://doi.org/10.1515/9783110884012	Tryon1995		This list comprises 1310 basic vocabulary items, extending the impactful 1110-items list by (:bib:Buck1949) by 200 items that are particularly relevant to Austronesian peoples.
\ No newline at end of file
+Tryon-1995-1310	Tryon, Darrell T.	1995		1310	questionnaire	English	Austronesian languages	https://doi.org/10.1515/9783110884012	Tryon1995		This list comprises 1310 basic vocabulary items, extending the impactful 1110-items list by (:bib:Buck1949) by 200 items that are particularly relevant to Austronesian peoples.
```

Comment:
@arubehn, we reference lists directly, if they are linked, by using `[1110-item list of Buck (1949)](:ref:Buck-1949-1110)`, if that list exists. You have `(xxx)` without the preceding `[xxx]` here anyway.